zbus/src/message.rs

```diff
@@ -196,6 +196,17 @@ impl Message {
         b.build(body)
     }
 
+    /// Create a message from bytes.
+    ///
+    /// The created message will have a receive sequence of 0.
```

Comment:
Kudos for documenting this. I think it would be good to:

1. say why sequence is `0`.
2. document `fds` parameter. @elmarco any suggestions for that?
3. document that the method only has `fds` parameter on unix platforms.

Reply:
thanks, that sounds good but i didn't mean just the general description of the paramter but more like what the parameter (could) imply for the user of this API. E.g if they construct a message from old saved message bytes from disk (like is the case for @SeaDve here) then they'll just pass an empty vec here. We need to tell that and when it woudl be empty and when it would be relevant.

Reply:
I think if you pass an empty vec and have UNIX_FD in the message, it will fail to deserialize. We can't really do anything "correct" in this case. It depends how the caller use the API.. but they can always pass /dev/null fds....

Reply:
true. The method is marked `unsafe` for good reasons anyway but we should document this.

Reply:
> Why sequence is 0.

I'm not sure exactly how to document this

Reply:
>>  Why sequence is 0.
>
> I'm not sure exactly how to document this

Let me do it for you then. :) Add this paragraph:

```rust
/// **Note:** Since the constructed message is not construct by zbus, the [receive sequence](`Message::recv_position`) is not applicable and hence set to `0`.
```

and remove the " with a receive sequence of 0" part from the summary (it's not the most important detail about this method).

P.S. I've forgotte the rustdoc markdown for linking to an identifier so please do check if the above linking works as expected.

Reply:
Thanks!